debugcc.c

```diff
@@ -207,32 +207,6 @@ static const struct debugcc_platform *find_platform(const char *name)
 	return NULL;
 }
 
-/**
```

Comment:
I can not say that it is unnecessary. I frequenty run it as `platform-debugcc` instead of `debugcc -p platform`.

BTW: is your commit not signed-of, or is it a Github hiding that bit of the commit message?

Reply:
Even better would be if `debugcc` could detect the platform and not have us accidentally map and read/write the wrong memory region for a different platform (resulting in an insta-crash usually).

I tend to do this a lot thanks to shell history.

Reply:
@lumag xkcd 1172 😛

@MarijnS95 should be very doable.. we can probably just read the compatible from /sys/firmware/devicetree